kythe/typescript/kythe.ts

```diff
@@ -0,0 +1,206 @@
+/**
+ * A VName (Vector Name) for a node in the Kythe schema consists of:
+ *  - `signature`: a unique, opaque signature for a node
+ *  - `corpus`: a collection of related files the node is defined in
+ *  - `root`: a label denoting a distinct subset of the corpus
+ *  - `path`: the relative path of the file containing the node
+ *  - `language`: programming language the node belongs to
+ */
+export interface VName {
```

Comment:
Shouldn't this be used by indexer.ts?

Reply:
Yes, going to add that in a separate PR.